editor/import/resource_importer_texture.cpp

```diff
@@ -586,6 +586,18 @@ Error ResourceImporterTexture::import(const String &p_source_file, const String
 				}
 			}
 		}
+
+		if (pack_channels == 3) { // BGRA mode, swapping B and R channels
+			const int height = target_image->get_height();
+			const int width = target_image->get_width();
+
+			for (int i = 0; i < width; i++) {
+				for (int j = 0; j < height; j++) {
+					const Color color = target_image->get_pixel(i, j);
+					target_image->set_pixel(i, j, Color(color.b, color.g, color.r));
```

Comment:
```suggestion
					target_image->set_pixel(i, j, Color(color, 1.0));
```
Equivalent and more compact

Reply:
Sorry, but no. Here we are swapping the channel order: to use BGRA instead of RGBA

Reply:
My bad misread